client/src/app/models/course.model.ts

```diff
@@ -0,0 +1,22 @@
+export interface Course {
+  id: string,
+  title: string,
+  tags: string[],
+  launchDate: Date | string,
+  status: string,
+  description: string,
+  duration: number,
+  lessonsCount: number,
+  containsLockedLessons: boolean,
+  previewImageLink: string,
+  rating: number,
+  meta: {
+    slug: string,
+    skills: string[],
+    courseVideoPreview: {
+      link: string,
+      duration: number,
+      previewImageLink: string,
+    }
+  }
```

Comment:
You can create a separate interface for meta as you use it in the Course and CourseDetail interfaces